pygmt/helpers/decorators.py

```diff
@@ -64,7 +64,7 @@
          """,
     "a": r"""
         aspatial : str
-            [*col*\ =]\ *name*\ [,...].
+            [[*col*\ =]\ *name*\ [,...]].
```

Comment:
Does it mean that we can use `aspatial=True` without giving string type arguments?

Reply:
I'll check

Reply:
Yes, `aspatial=True` does work. I will update accordingly. An example:
```
import pygmt
pygmt.info(table='points.txt',aspatial=True)
```
where points.txt contains:
```
# @VGMT1.0 @GPOINT @Nname|depth|id
# @Tstring|double|integer
# @R178.43/178.5/-57.98/-34.5
# @Je4326
# @Jp"+proj=longlat +ellps=WGS84 +datum=WGS84+no_defs"
# FEATURE_DATA
# @D"point 1"|-34.5|1
178.5 -45.7
# @D"Point 2"|-57.98|2
178.43 -46.8
```

Reply:
I am looking at a case in GMT core right now where -a (no args, the equivalent of your True) only prints out the aspatial values and not the coordinates.  The intent is that it should work (and it used to).

Reply:
Yes the output currently includes both spatial and spatial info, as you noted in the other PR. So, a bug.
`'Aspatial columns: name[string] depth[double] id[integer]\npoints.txt: N = 2 <178.43/178.5> <-46.8/-45.7> <-57.98/-34.5> <1/2>\n'`

Reply:
I added that `aspatial` can handle `bool` as an input, even though it doesn't work quite right because it is still an option.